Automattic/simplenote-ios
# Setup: Renaming config.plist

### Details:
Although we've got the `config.plist` in **.gitignore**, since we've got a sample file already pushed, all of the local changes are immediately picked up by git, and marked as `diff`.

In this PR we're renaming `config.plist` > `config-demo.plist`, and adding an extra step in the Readme.

@frosty may i bug you with a small PR?
Thanks in advance!

cc @roundhill 

### Testing:
1. Checkout this branch
2. Verify that it won't build right away
3. Rename `Simplenote/config-demo.plist` > `Simplenote/config.plist`
4. Verify that **git status** shows no changes
